pkgs/applications/audio/pmidi/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, alsaLib 
+, version ? "1.7.1"
+, sourceSha256 ? "051mv6f13c8y13c1iv3279k1hhzpz4fm9sfczhgp9sim2bjdj055"
+}:
+stdenv.mkDerivation {
+  name = "pmidi-${version}";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/pmidi/${version}/pmidi-${version}.tar.gz";
+    sha256 = sourceSha256;
+  };
+
+  buildInputs = [ alsaLib ];
+
+  meta = with stdenv.lib; {
+    description = "";
+    maintainers = with maintainers; [ lheckemann ];
```

Comment:
Homepage: http://www.parabola.me.uk/alsa/pmidi.html
Description?
Please set `license = licenses.unfree` and mentiod that no license was specified.

Reply:
Whoops, my mistake! I thought I'd finished that stuff. It's in fact GPLv2 (see contents of source tarball). Thanks for the pointers.